.travis.yml

```diff
@@ -5,3 +5,9 @@ node_js:
   - 6
   - 8
   - 9
+
+jobs:
+  include:
+    - stage: coverage
+      node_js: 8
```

Comment:
should try to keep this aligned with current LTS version

Reply:
To automatically stay updated with the latest LTS version, you can use `'lts/*'`, as described [here](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions).